index.html

```diff
@@ -102,63 +102,133 @@ <h1 id="title">Dubbing and Audio description Profiles of TTML2</h1>
-        It is the result of the audio rendition of one or more <a>descriptions</a>
-        mixed with the audio associated with the programme prior to any mixing with audio description
-        (sometimes referred to as <dfn>main programme audio</dfn>),
-        at moments when this does not clash with dialogue, to deliver an <dfn>audio description mixed audio track</dfn>.
-        A <dfn>description</dfn> is a set of words that describe an aspect of the programme presentation,
-        suitable for rendering into audio by means of vocalisation and recording
-        or used as a text alternative source for text to speech translation, as defined in [[WCAG20]].
-        More information about what <a>audio description</a> is and how it works can be found at [[WHP051]].
-      </p>
-
-      <p class="issue" data-number="28"></p>
-      <p>This specification defines DAPT, a TTML-based format for the exchange of timed text content among authoring and prompting tools in the localization and <a>audio description</a> pipeline.
-        A DAPT file is designed to carry pertinent information for dubbing or <a>audio description</a> such as type of script, dialogue, descriptions, timing, metadata, original language text, transcribed text, language information, and to be extensible for future annotations.
-        This specification first defines the data model (see [[[#data-model]]]) for DAPT scripts and then its representation as a TTML document with restrictions (see [[[#ttml-format]]]).</p>
+      <section id="intro-overview">
+        <h3>Transcripts and Scripts</h3>
+        <p>A <dfn>transcript</dfn> is a text representation of <i>pre-existing</i> media in another form,
+          for example the dialogue in a video;
+          a <dfn>script</dfn> is a text representation of the <i>intended</i> content of media prior to its creation,
+          for example to guide an actor in recording an audio track.
+          Within this specification the term <a>DAPT script</a> is used generically to refer to both <a>transcripts</a> and <a>scripts</a>.
+          <a>DAPT Scripts</a> consist of timed text and associated metadata,
+          such as the character speaking.
+        </p>
+        <p>In dubbing workflows, a <a>transcript</a> is generated and translated to create a <a>script</a>.
+          In audio description workflows, a <a>transcript</a> describes the video image,
+          and is then used directly as a <a>script</a> for recording an audio equivalent.
+        </p>
+        <p>DAPT is a TTML-based format for the exchange of <a>transcripts</a> and <a>scripts</a>
+          (i.e. <a>DAPT Scripts</a>)
+          among authoring, prompting and playback tools in the localization and <a>audio description</a> pipelines.
+          A <dfn>DAPT document</dfn> is a serialisable form of a <a>DAPT Script</a> designed to carry pertinent information for dubbing or <a>audio description</a>
```

Comment:
shouldn't this be "serializable" (z, American English?)

Reply:
According to https://www.w3.org/Guide/manual-of-style/#Spelling yes. Also, after publication on /TR we can run the W3C spell checker, and fix any errors discovered there. Fixed in 82787db307f7b8e0c71a8bdb2348a015ae067f23